.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "snapshot-spaces"]
+	path = snapshot-spaces
+	url = https://github.com/snapshot-labs/snapshot-spaces
```

Comment:
Would be better if we can move verified.json to sidekick so we don't need to redeploy the hub everytime there is a change there.

Reply:
Data from sidekick isn't ready yet, but will be soon hopefully

Reply:
Can we remove this submodule now? @ChaituVR